src/components/utils/onfidoApi.ts

```diff
@@ -174,9 +174,13 @@ export const uploadLiveVideo = (
     switchSeconds: challenge_switch_at,
   } = challengeData || {}
 
+  // NOTE: important for automation - language_code string format must be BCP-47 IIRC, i.e. "en-US"
+  const languageCodeForApi = language && language.replace('_', '-')
```

Comment:
How about just sending the first two characters given we are not really using the region part?

Reply:
Good point. Have revisited the original Slack thread where this formatting issue was raised & we can definitely do that. Will leave the note with a bit more detail about the supported formats so we don't accidentally impact automation again.